functions/src/common/broadcast-survey-update.ts

```diff
@@ -0,0 +1,28 @@
+/**
+ * @license
+ * Copyright 2024 The Ground Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as admin from 'firebase-admin';
+
+export async function broadcastSurveyUpdate(topic: string): Promise<string> {
+  // Messages are sent without a payload so that they can collapsed. Collapsible
```

Comment:
Please add tsdoc for this method: 
"Sends an empty message to clients subscribed to the specified topic."

Also please move this block into the tsdoc.